src/interfaces/assistants_web/src/components/Shared/Modal/Modal.tsx

```diff
@@ -128,38 +52,36 @@ export const Modal: React.FC<ModalProps> = ({
             leaveTo="opacity-0 scale-90"
           >
             {/* Container to center the panel */}
-            <div className="absolute top-0 flex min-h-full w-full items-center justify-center overflow-auto p-4">
+            <div
+              className={cn(
+                'absolute top-0 flex min-h-full w-full items-center justify-center overflow-auto p-4 md:w-modal'
+              )}
+            >
               {children && (
-                <Dialog.Panel className={cn(panelStyle({ kind, className: panelClassName }))}>
-                  {!hideCloseButton && title && (
+                <DialogPanel
+                  className={cn(
+                    'h-fit max-h-modal overflow-y-auto rounded-lg border px-5 py-7 md:px-10 md:py-14',
+                    'flex flex-col gap-y-8',
+                    'border-marble-900 dark:border-volcanic-500',
+                    'bg-mushroom-950 dark:bg-volcanic-200'
+                  )}
+                >
+                  {title && (
                     <header>
-                      {!hideCloseButton && (
-                        <CloseButton className={cn(closeButtonStyle({ kind }))} onClose={onClose} />
-                      )}
                       {title && (
-                        <Dialog.Title as="div" className="flex-0">
-                          {typeof title === 'string' ? (
-                            <Text
-                              as="h5"
-                              className={cn(titleStyle({ kind }))}
-                              styleAs={isCitation ? 'label' : 'h5'}
```

Comment:
we don't have modal citations in mobile?

Reply:
not shown in designs, but it's easy to change this with specs